comp/dogstatsd/pidmap/state.go

```diff
@@ -1,9 +1,4 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-2020 Datadog, Inc.
-
-package replay
```

Comment:
This should probably live in a `pidmapimpl` package. We have been trying to keep the component interface and implementations separate